rov/movement/controls/Base_Control_Handler.py

```diff
@@ -0,0 +1,22 @@
+from abc import ABCMeta, abstractmethod
+
+class BaseControlHandler(object):
```

Comment:
What is the point of this class if we also have ControlHandler? Get rid of one (and keep the name BaseControlHandler for the one you keep).

Reply:
`BaseControlHandler` is an abstract class. So any control handler should implement this. `ControlHandler` is supposed to be some implementation of it, but there can be many. I don't see exactly where it's going to be used though.